psydac/api/discretization.py

```diff
@@ -204,7 +206,116 @@ def solve(self, **kwargs):
 
         return driver_solve(self.linear_system, **settings)
 
+#==============================================================================
+class DiscreteDerham(BasicDiscrete):
```

Comment:
I know that the documentation is very scarce in this file, but can we add a docstring to class declarations at least?

Reply:
done